src/main/resources/templates/fragments/login_navigation.html

```diff
@@ -42,14 +24,15 @@
 						<li class="scroll"><a href="../about.html">About</a></li>
 						<li class="scroll"><a href="../event.html">Events</a></li>
 						<li class="scroll"><a href="../plan.html">Features</a></li>
+						<li class="scroll"><a href="../tools.html">Tools</a></li>
 						<li class="scroll"><a href="../pricing.html">Pricing</a></li>
-						<li class="scroll"><a href="research.html">Research</a></li>
+						<!--<li class="scroll"><a href="research.html">Research</a></li>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)